### ISSUE TITLE
Benchmark performance before / after enabling new plugin for header level corrections

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#12345]
## Description
#12342 will implement a new plugin that will modify static HTML files after content-build to correct header-level violations. This plugin may have performance implications, and we must benchmark before shipping. 

## Tasks
* Benchmarking of build times before and after will be required to make ensure we aren't adding a significant amount of build time

## Performance threshhold / criteria
* TBD - eng input needed
  * Q: How many runs are required to confirm a baseline vs. new data? 
  * Q: Can this be automated / written in a way that it can be re-used in future? How much effort does that add? 
  * Q: What is an acceptable level of performance hit to merge the plugin?

## Acceptance Criteria
- [x] Performance timing of content-build is documented in ticket before and after enabling the new plugin. 
- [x] Threshold for performance diff is met / not exceeded.
  - [ ] If exceeded, raise to PM and reopen / move #12342 back to To Do, with notes.
  - [x] If perf criteria are met, indicate on #12342 / PR that it's cleared to merge
- [x] Inform Dave C about benchmark results

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [X] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by davidmpickett** [comment]
Just wanted to say that this ticket number is super cool

**Comment by jtmst** [comment]
The average performance hit when built on local with the new plugin is 2 minutes, 21 seconds.    This will be able to be monitored more closely with Datadog once its on prod.  The current thinking is that 2:21 would be acceptable, but the tradeoff should be presented to Dave C.  Slack convo on the topic here: https://dsva.slack.com/archives/C52CL1PKQ/p1677620105583029

**Comment by jtmst** [comment]
In order to get a better idea of performance impact, I tested locally by comparing build times with the plugin disabled vs with the plugin enabled.  I ran each 5 times and took an average:

With plugin:
17.05
14.6
14.6
15.04
14.58
——
15.17

Without plugin:
14.75
14.8
14.16
14.3
14.4
——
14.48

This demonstrates a 4.76% increase.  This was conducted locally on a macbook, but gives us some kind of baseline of what we might expect to happen on prod.
